.dockerignore

```diff
@@ -0,0 +1,72 @@
+# Miscellaneous
+*.class
+*.log
+*.pyc
+*.swp
+*.snap
+.DS_Store
+.local/
+.atom/
+.buildlog/
+.history
+.svn/
+prime
```

Comment:
Doesn't it make more sense to only list things we have? It's unlikely that we add a .atom or .svn directory in the future